JustSaying.AwsTools/SqsNotificationListener.cs

```diff
@@ -88,7 +80,7 @@ public SqsNotificationListener WithMessageProcessingStrategy(IMessageProcessingS
             }
 
             Subscribers.Add(new Subscriber(typeof(T)));

```

Comment:
Not entirely related, but why is this list of subscribers populated? Does any other code read it? If not, could we delete it without anything breaking?